android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -702,10 +703,11 @@ public static void setBitmapResource(ExecutorService executor, Handler handler,
         }
         executor.execute(() -> {
             Uri logoFileUri = Uri.parse(iconPath);
-            int resizeFactor = 110;
+            int resizeFactorTemp = 110;
             if (textView != null) {
-                resizeFactor = 70;
+                resizeFactorTemp = 70;
             }
+            final int resizeFactor = resizeFactorTemp;
```

Comment:
we can simplify this assignment without creating a temp variable.

Reply:
final can not be set to a diff value once it's set, in the above case it could be 110 or 70.